content/en/blog/_posts/2022-09-26-csi-migration-status.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: blog
+title: "Kubernetes 1.25: Kubernetes In-Tree to CSI Volume Migration Status Update"
+date: 2022-09-26
+slug: storage-in-tree-to-csi-migration-status-update-1.25
+---
+
+**Author:** Jiawei Wang (Google)
+
+The Kubernetes in-tree storage plugin to [Container Storage Interface (CSI)](/blog/2019/01/15/container-storage-interface-ga/) migration infrastructure has already been [beta](/blog/2019/12/09/kubernetes-1-17-feature-csi-migration-beta/) since v1.17. CSI migration was introduced as alpha in Kubernetes v1.14.
+Since then, SIG Storage and other Kubernetes special interest groups are working to ensure feature stability and compatibility in preparation for CSI Migration feature to go GA.
+
+SIG Storage is excited to announce that the core CSI Migration feature is **generally available** in Kubernetes v1.25 release!
+
+SIG Storage team had a blog post in v1.23 for [CSI Migration status update](https://kubernetes.io/blog/2021/12/10/storage-in-tree-to-csi-migration-status-update/) which discussed about the CSI migration status for each storage  drivers. It has been a while and this article is intended to give a latest status update on each storage driver for their CSI Migration status in Kubernetes v1.25.
```

Comment:
```suggestion
SIG Storage team had a blog post in v1.23 for [CSI Migration status update](https://kubernetes.io/blog/2021/12/10/storage-in-tree-to-csi-migration-status-update/) which discussed the CSI migration status for each storage driver. It has been a while and this article is intended to give a latest status update on each storage driver for their CSI Migration status in Kubernetes v1.25.
```

Reply:
“SIG Storage team” is mildly tautological. @Jiawei0227 you could start this paragraph with “SIG Storage wrote a blog post” and leave out the “team”. Remember that the G in SIG is almost a synonym for “team”.

If _keeping_ the word “team”, that paragraph should begin with “The”.

Reply:
Done